.clusterfuzzlite/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM gcr.io/oss-fuzz-base/base-builder@sha256:14b332de0e18683f37386eaedbf735bc6e8d81f9c0e1138d620f2178e20cd30a
+ENV MERGE_WITH_OSS_FUZZ_CORPORA yes
```

Comment:
Looks like this `ENV` syntax is deprecated and might be removed in the future (https://docs.docker.com/engine/reference/builder/#env "Alternative syntax"). Not sure if it matters here though :-)

Reply:
Thanks! I'll change that

Reply:
Done. It's being tested https://github.com/evverx/systemd/runs/4998101837?check_suite_focus=true